api/src/main/java/com/alibaba/nacos/api/common/Constants.java

```diff
@@ -24,172 +24,174 @@
  * @author Nacos
  */
 public class Constants {
-    
+
```

Comment:
Please do not change the indent.

Please use nacos' code style to reformat codes.